test/unit/components/provider.tsx

```diff
@@ -1,9 +1,12 @@
 import * as React from 'react';
 import {mount} from 'enzyme';
+import areIntlLocalesSupported from 'intl-locales-supported';
 import IntlProvider, {Props} from '../../../src/components/provider';
 import {WithIntlProps} from '../../../src/components/injectIntl';
 import {IntlShape} from '../../../src/types';
-import withIntl, {Provider} from '../../../src/components/injectIntl';
+import withIntl from '../../../src/components/injectIntl';
+
+jest.mock('intl-locales-supported');
```

Comment:
This is necessary otherwise the test fails because the intl polyfill (for the tests) returns `false` for some of the intl contructors as they don't support languages other than `en`.